zeus/tasks/delete_repo.py

```diff
@@ -0,0 +1,25 @@
+from uuid import UUID
+
+from flask import current_app
+
+from zeus import auth
+from zeus.config import celery, db, redis
+from zeus.constants import Permission
+from zeus.models import Repository, ItemOption, RepositoryAccess
+
+
+@celery.task(name="zeus.delete_repo")
+def delete_repo(repo_id: UUID):
+    auth.set_current_tenant(auth.Tenant(access={repo_id: Permission.admin}))
```

Comment:
Do we actually need this? I don't fully understand the purpose of this tenant magic.